src/components/Checklist/Checklist.tsx

```diff
@@ -1,75 +1,71 @@
+import { fieldworkChecklist } from "./md-in-js/fieldworkChecklist";
 import styles from "./Checklist.module.css";
 
 const Checklist: React.FC = () => {
   return (
-    <IonAccordionGroup>
-      <div className={styles.header}>
-        <span className={styles.title}>{soilPackageChecklist.header}</span>
-        <IonIcon
-          id="hover-trigger-soil"
-          icon={informationCircleOutline}
-          size="small"
-          color="primary"
-        ></IonIcon>
-        <IonPopover trigger="hover-trigger-soil" triggerAction="hover">
-          <IonContent class="ion-padding">
-            {soilPackageChecklist.info.split("\n").map(function (item, index) {
-              return (
-                <span key={index}>
-                  {item}
-                  <br />
-                </span>
-              );
-            })}
-          </IonContent>
-        </IonPopover>
+    <div>
+      <div className={styles.info}>
+        <Markdown>{fieldworkChecklist.info}</Markdown>
       </div>
-
       {/* Render each section in its own accordion. */}
-      {soilPackageChecklist.sections.map((s) => (
-        <IonAccordion key={s.title}>
-          <IonItem slot={"header"}>
-            <IonLabel>{s.title}</IonLabel>
-          </IonItem>
-          <div className={"ion-padding"} slot={"content"}>
+      {fieldworkChecklist.sections.map((s) => (
+        <div key={s.title}>
+          <SectionHeader>{s.title}</SectionHeader>
+          <div slot={"content"}>
+            <Markdown className={styles.info}>{s.info}</Markdown>
             <Markdown
               // Map Markdown elements to React elements.
               components={{
                 ul: (props) => <IonList>{props!.children}</IonList>,
-                li: (props) => (
-                  <IonItem className={styles.listItem} lines={"none"}>
-                    <IonIcon
-                      size={"small"}
-                      slot={"start"}
-                      icon={checkmarkCircle}
-                      color={"primary"}
-                    />
-                    <span>{props!.children}</span>
-                  </IonItem>
-                ),
+                li: (props) => {
+                  // use different icon for the nested list item
+                  const match = /^NESTEDLISTITEM/.exec(
+                    props!.children?.toString() || "",
+                  );
+                  return match ? (
+                    <IonItem className={styles.listItem} lines={"none"}>
+                      <IonIcon
+                        className={styles.listIcon}
+                        size={"small"}
+                        slot={"start"}
+                        icon={checkmarkCircleOutline}
+                        color={"primary"}
+                      />
+                      <IonLabel className={styles.listLabel}>
+                        {props!.children
+                          ?.toString()
+                          .replace(/NESTEDLISTITEM /, "")}
```

Comment:
This has the same effect as the original version, but I prefer it because it more explicitly shows that we are targeting the occurrence of `NESTEDLISTITEM` that is at the beginning of the string.

```suggestion
                          .replace(/^NESTEDLISTITEM /, "")}
```